scripts/superdesk/editor2/editor.js

```diff
@@ -562,6 +562,13 @@ function SdTextEditorBlockEmbedController($timeout, editor, renditions) {
                 vm.onBlockChange();
             });
         },
+        isEditable: function(picture) {
+            if (picture._type === 'externalsource') {
+                return false;
+            } else {
+                return true;
+            }
```

Comment:
You can replace all of this function with `return picture._type === 'externalresource'`